src/core/qgsfeaturefiltermodel.h

```diff
@@ -196,8 +196,9 @@ class CORE_EXPORT QgsFeatureFilterModel : public QAbstractItemModel
      * Allows specifying one value that does not need to match the filter criteria but will
      * still be available in the model as NULL value(s).
      * \since QGIS 3.10
+     * \deprecated since QGIS 3.12 use setExtraIdentifierValues( QVariantList() ) instead
      */
-    void setExtraIdentifierValuesToNull();
+    Q_DECL_DEPRECATED void setExtraIdentifierValuesToNull() SIP_DEPRECATED;
```

Comment:
Is it really necessary to deprecate this method? I mean, the way you fix it should work now and it doesn't harm to keep it.

Same comment worth for setIdentifierValuesToNull

Reply:
Okay, I removed it again. And used always this function on setting to null.